tst/wedderga07.tst

```diff
@@ -15,9 +15,9 @@ gap> G:=SmallGroup(48,15);
 gap> R:=GroupRing(Rationals,G);
 <algebra-with-one over Rationals, with 5 generators>
 gap> WedderburnDecompositionInfo(R);
-[ [ 1, Rationals ], [ 1, Rationals ], [ 1, Rationals ], [ 1, Rationals ], 
+[ [ 1, Rationals ], [ 1, Rationals ], [ 1, Rationals ], [ 1, Rationals ],
```

Comment:
I will backport these changes to examples in xml files while rebasing (I know that for you it is difficult on Windows, so fine!)

Reply:
Actually, these are whitespace changes - nothing to backport then!